### DISCUSSION TITLE
I can't push my commit in Android Studio

### DISCUSSION THREAD

**Discussion by Archangel354** [community/community#45735] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have writing code for an online Android class but as of a week ago I can no longer push my commits to github.   When I push it just sits there saying "Pushing..."

**Comment by jge162** [comment]
@Archangel354 You need to check the following conditions

- Network connectivity issues, check your internet connection.
- Git repository connection issues, try re-adding the remote repository.
- Authentication failure, ensure your credentials are correct and re-authenticate.
- Large files in your commit that exceed Github's limit, try using Git Large File Storage (LFS) to store large files.
- Git configuration problems, try removing the Git configuration file and re-configuring Git.

Not sure which one of these conditions is your issue but one of them is likely to be the cause.

**Comment by cehfisher** [comment]
Hi @Archangel354 - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to a different category? Thanks!
